static/src/javascripts/projects/common/modules/atoms/youtube.js

```diff
@@ -252,55 +263,84 @@ const STATES = {
     PAUSED: onPlayerPaused,
 };
 
-const getIFrameBehaviour = (atomId: string): IFrameBehaviour => {
-    const isAutoplayBlockingPlatform = () => isIOS() || isAndroid();
+const getIFrameBehaviourConfig = (
+    iframe: HTMLIFrameElement
+): IFrameBehaviourConfig => {
+    const isAutoplayBlockingPlatform = isIOS() || isAndroid();
 
-    const isInternalReferrer = (): boolean => {
+    const isInternalReferrer = ((): boolean => {
         if (config.get('page.isDev')) {
             return document.referrer.indexOf(window.location.origin) === 0;
         }
 
         return document.referrer.indexOf(config.get('page.host')) === 0;
-    };
+    })();
 
-    const isMainVideo = (): boolean =>
-        (players[atomId].iframe &&
-            !!players[atomId].iframe.closest(
-                'figure[data-component="main video"]'
-            )) ||
+    const isMainVideo =
+        (iframe && !!iframe.closest('figure[data-component="main video"]')) ||
         false;
 
-    const flashingElementsAllowed = (): boolean =>
-        accessibilityIsOn('flashing-elements');
+    const flashingElementsAllowed = accessibilityIsOn('flashing-elements');
 
-    const isVideoArticle = (): boolean =>
+    const isVideoArticle =
         config.get('page.contentType', '').toLowerCase() === 'video';
 
-    const isFront = () => config.get('page.isFront');
+    const isFront = config.get('page.isFront');
     const isUSContent =
         config.get('page.productionOffice', '').toLowerCase() === 'us';
 
     const isPaidContent = config.get('page.isPaidContent');
+
+    return {
+        isAutoplayBlockingPlatform,
+        isInternalReferrer,
+        isMainVideo,
+        flashingElementsAllowed,
+        isVideoArticle,
+        isFront,
+        isUSContent,
+        isPaidContent,
+    };
+};
+
+const getIFrameBehaviour = (
+    iframeConfig: IFrameBehaviourConfig
+): IFrameBehaviour => {
+    let iFrameBehaviour;
+
+    const {
+        isAutoplayBlockingPlatform,
+        isInternalReferrer,
+        isMainVideo,
+        flashingElementsAllowed,
+        isVideoArticle,
+        isFront,
+        isUSContent,
+        isPaidContent,
+    } = iframeConfig;
+
     const isUsPaidContentVideo =
         isUSContent &&
         isPaidContent &&
-        ((isVideoArticle() && isMainVideo()) || isFront()) &&
-        flashingElementsAllowed();
+        ((isVideoArticle && isMainVideo) || isFront) &&
+        flashingElementsAllowed;
+
     if (isUsPaidContentVideo) {
-        return {
+        iFrameBehaviour = {
             autoplay: isUsPaidContentVideo,
-            mutedOnStart: isUsPaidContentVideo,
+            mutedOnStart: isUsPaidContentVideo && isAndroid(),
+        };
+    } else {
```

Comment:
We can actually remove the else block, which should help with making the linter happy. This used to be part of the linting rules for frontend aaaages ago, but may have changes so feel free to ignore 😄 

```js
if (isUsPaidContentVideo) {
    return {
        autoplay: isUsPaidContentVideo,
        ...
    }

return {
    autoplay: 
    ...
}
```